src/responsive-manager.ts

```diff
@@ -0,0 +1,83 @@
+import { PixelRatio } from 'react-native';
+import { DIMENSIONS_RATIO, SCREEN_HEIGHT, SCREEN_WIDTH } from './constants';
+
+interface IDeviceParams {
+  width: number;
+  height: number;
```

Comment:
we can add posibility set width & height OR deviceName, e.g. IPhone15 with predefined dimensions

Reply:
I think that this doesn't make sense, since these values are set only once and it is impossible to provide for all the options for device sizes that designers use. In any case, we will have to constantly update the list of sizes, which doesn't look very convenient.